pkg/testing/v1alpha1/revision.go

```diff
@@ -96,63 +96,64 @@ func WithRevStatus(st v1alpha1.RevisionStatus) RevisionOption {
 
 // MarkActive calls .Status.MarkActive on the Revision.
 func MarkActive(r *v1alpha1.Revision) {
-	r.Status.MarkActive()
+	r.Status.MarkActiveTrue()
```

Comment:
Is there a reason that we are deviating the names of `MarkXxx` functions from their respective revision status methods?

Reply:
We tied the function names to the `ConditionType` plus the status they can have (`True`, `False` or `Unknown`) and decided to keep the naming verbose (so `MarkActiveTrue()` rather than `MarkActive()`). see this https://github.com/knative/serving/pull/5804#discussion_r334759658